app/src/main/res/values/strings.xml

```diff
@@ -43,4 +43,32 @@
     <string name="coming_soon">Coming soon</string>
     <string name="get_started">Get started</string>
     <string name="change_mastercopy">Change mastercopy</string>
+
+    <string name="settings_tab_safe">SAFE SETTINGS</string>
+    <string name="settings_tab_app">APP SETTINGS</string>
```

Comment:
These shouldn't be all uppercase here. I think that tab names are upper case by default. Otherwise, I would call `String.toUpperCase()` rather than having an assert fully in upper case.

Reply:
We should try to use strings as is. This will allow us to replace our strings with resources downloaded from localized, which we will use at the latter point. Otherwise there would be too much ambiguity regarding lower/upper case, capital letters, etc.